gcp/table_gcp_monitoring_alert_policy.go

```diff
@@ -0,0 +1,198 @@
+				Type:        proto.ColumnType_JSON,
+			},
+
+			// standard steampipe columns
+			{
+				Name:        "tags",
+				Description: ColumnDescriptionTags,
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("UserLabels"),
+			},
+			{
+				Name:        "title",
+				Description: ColumnDescriptionTitle,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DisplayName"),
+			},
+			{
+				Name:        "akas",
+				Description: ColumnDescriptionAkas,
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getMonitoringAlertPolicyAka,
+				Transform:   transform.FromValue(),
+			},
+
+			// standard gcp columns
+			{
+				Name:        "location",
+				Description: ColumnDescriptionLocation,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromConstant("global"),
+			},
+			{
+				Name:        "project",
+				Description: ColumnDescriptionProject,
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getProject,
+				Transform:   transform.FromValue(),
+			},
+		},
+	}
+}
+
+//// LIST FUNCTION
+
+func listMonitoringAlertPolicy(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	// Create Service Connection
+	service, err := MonitoringService(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+
+	// Get project details
+	projectData, err := activeProject(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+	project := projectData.Project
+
+	resp := service.Projects.AlertPolicies.List("projects/" + project)
+
+	if err := resp.Pages(ctx, func(page *monitoring.ListAlertPoliciesResponse) error {
+		for _, alertPolicy := range page.AlertPolicies {
+			d.StreamListItem(ctx, alertPolicy)
+		}
+		return nil
+	}); err != nil {
+		return nil, err
+	}
+
+	return nil, err
+}
+
+//// HYDRATE FUNCTIONS
+
+func getMonitoringAlertPolicy(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("getMonitoringAlertPolicy")
+
+	// Get project details
+	projectData, err := activeProject(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+	project := projectData.Project
+	name := d.KeyColumnQuals["name"].GetStringValue()
+
+	// Create Service Connection
+	service, err := MonitoringService(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+
+	req, err := service.Projects.AlertPolicies.Get("projects/" + project + "/alertPolicies/" + name).Do()
+	if err != nil {
+		return nil, err
+	}
+
+	// If the name has been passed as empty string, API does not return any error
+	if len(req.Name) < 1 {
+		return nil, nil
+	}
+
+	return req, nil
+}
+
+func getMonitoringAlertPolicyAka(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	alertPolicy := h.Item.(*monitoring.AlertPolicy)
+	
+	akas := []string{"gcp://monitoring.googleapis.com/" + alertPolicy.Name}
+
+	return akas, nil
+}
```

Comment:
Do we need hydrate function to build the aka? or, we can use transform function?